frontend/src/app/modules/result-selection/result-selection.component.html

```diff
@@ -0,0 +1,4 @@
+<ng-container *ngIf="currentChart === 'TimeSeries' || 'PageAggregation'">
+  <osm-result-selection-time-frame [currentChart]="currentChart"></osm-result-selection-time-frame>
+  <osm-result-selection-job-group [currentChart]="currentChart"></osm-result-selection-job-group>
```

Comment:
```suggestion
  <osm-result-selection-job-group></osm-result-selection-job-group>
```
The currentChart is not needed in the component.